src/core/mark/legend.ts

```diff
@@ -15,7 +14,7 @@ type LegendOffset = { offsetRight: number };
 export function drawColorLegend(
     HGC: { libraries: Libraries },
     trackInfo: any,
-    tile: Tile,
+    _tile: unknown,
```

Comment:
This is just to fix our tests (which provide a tile as a `null` value). These functions should ideally define an interface which does not include `tile` as property,

```typescript
interface DrawColorLegendOptions {
  trackInfo: any;
  model: GoslingModel;
  theme: Required<CompleteThemeDeep>;
}

export function drawColorLegend(HGC, opts: DrawColorLegendOptions);

drawColorLegend(HGC, { trackInfo, model, theme }) // ok
drawColorLegend(HGC, { trackInfo, model, theme, track }) // ok
drawColorLegend(HGC, { trackInfo, theme, track }) // error
```

Reply:
Maybe we gotta remove `_tile` from the argument since it seems like the variable is not used inside the functions.

Reply:
That was my thought, but I wasn't sure if other functions had a similar signature (ie, `draw*()`) and it would be useful to allow for `tile` to be passed somehow when calling.

Reply:
something for a follow up PR I think.